.github/agents/streamhub-state.agent.md

```diff
@@ -22,13 +22,36 @@ You specialize in:
 
 Any StreamHub maintaining stateful fields beyond simple cache lookups MUST override RollbackState(DateTime timestamp).
 
-### Common Scenarios Requiring RollbackState
+### Common Scenarios Requiring RollbackState (Incremental Pattern)
 
 1. **Rolling windows** - RollingWindowMax/Min must be rebuilt from cache
 2. **Buffered historical values** - Raw buffers (e.g., K buffer in Stoch) must be prefilled
 3. **Running totals/averages** - EMA state, Wilder's smoothing must be recalculated
 4. **Previous value tracking** - _prevValue,_prevHigh, etc. must be restored
 
+### Scenarios Requiring RollbackState (Repaint from Pivot Pattern)
```

Comment:
Since this agent is used for all indicators, this variant should be described as repainting only the minimally needed history, which can vary for each uniquely repainting indicator, which is why we need the option to override.  Pivot point in zig zag is just an example and is only appropriate for zig zag.